doc/source/glossary.rst

```diff
@@ -330,6 +542,32 @@ Glossary
        >>> x[:, 1]
        array([2, 4])
 
+   stride
+
+      Physical memory is one-dimensional; ``stride`` maps an index in an
```

Comment:
```suggestion
      Physical memory is one-dimensional; a ``stride`` maps an index in an
```

Reply:
reworked this section differently, please re-review